creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -119,6 +119,58 @@ spec:
       networkName: ovs-vlan-100
 ----
 
+It is also possible to define a multus network as the default pod network with
+https://github.com/intel/multus-cni[Multus]. A version of multus after this
+https://github.com/intel/multus-cni/pull/174[Pull Request] is required
+(currently master).
+
+
+*Note the following:*
+
+* A multus default network and a pod network type are mutually exclusive.
+* The virt-launcher pod that starts the VMI will *not* have the pod network
+configured.
+* The multus delegate chosen as default *must* return at least one IP address.
+
+
+Create a `NetworkAttachmentDefinition` with IPAM.
+
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan-test
+spec:
+  config: '{
+      "type": "macvlan",
+      "master": "eth0",
+      "mode": "bridge",
+      "ipam": {
+        "type": "host-local",
+              "subnet": "10.250.250.0/24"
```

Comment:
@greg-bock can you please take a look here https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local

I think the `subnet` needs to by inside a `ranges` list

Reply:
> Previous versions of the host-local allocator did not support the ranges property, and instead expected a single range on the top level. This is deprecated but still supported.

I wanted to keep the example as small as possible.